app-editors/kakoune/kakoune-0_pre20161111.ebuild

```diff
@@ -42,6 +42,7 @@ src_configure() {
 	append-libs $($(tc-getPKG_CONFIG) --libs ncursesw)
 	export CXX=$(tc-getCXX)
 	export debug=$(usex debug)
+	export static=$(usex static)
 	S="${WORKDIR}/${P}/src"
 }
 
```

Comment:
the `src_install` command is not Prefix-aware, please make Prefix-aware (`${EPREFIX}`)